sirepo/template/radia.py

```diff
@@ -1389,9 +1564,32 @@ def _update_geom_obj(o, delim_fields=None, **kwargs):
     return o
 
 
-def _delim_string(val=None, default_val=None):
-    d = default_val if default_val is not None else []
-    return sirepo.util.to_comma_delimited_string(val if val is not None else d)
+def _update_jay(o, beam_axis, height_axis, **kwargs):
+    return _update_jay_points(_update_geom_obj(o, **kwargs), beam_axis, height_axis)
+
+
+def _update_racetrack(o, _, __, **kwargs):
+    return _update_geom_obj(o)
+
+
+def _update_stemmed_points(o, beam_axis, height_axis):
+    w, h, b, c, s = _stemmed_geometry(o, beam_axis, height_axis)
+
+    # start with arm top, stem left - then reflect across centroid axes as needed
```

Comment:
I think using named functions would be clearer than a comment and could reduce some of the repeated code.

Reply:
agree, see recent comment